server/app/services/export/CsvExporterService.java

```diff
@@ -144,6 +121,11 @@ private CsvExportConfig generateCsvConfig(
 
     ImmutableList<ApplicantQuestion> sortedUniqueQuestions =
         uniqueQuestions.values().stream()
+            // TODO: This sorts the paths lexicographically, so "household members[10] - name" is
+            // sorted above "household members[1] - name".
+            // It should be possible to write a comparator that iterativly compares segments of
+            // the Path, so that nested repeated questions are sorted correctly, but I don't have
+            // time at the moment.
```

Comment:
This is not a regression or change from how repeated questions are currently sorted, but it is annoying.